spec/mailers/previews/welcome_email_preview.rb

```diff
@@ -0,0 +1,12 @@
+# Preview all emails at http://localhost:3000/rails/mailers/welcome_email
```

Comment:
This is where you can preview the email. And here is a screenshot:

![Uploading Screen Shot 2015-05-03 at 5.10.13 PM.png…]()